### DISCUSSION TITLE
The longest war in history went for how many years?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#689] [Galaxy Trivia] [answered]
*answer in hex*

**Comment by SANKARAMDAS** [comment, marked as answer]
The Reconquista war- 781 years (approx)
Hex number (3 digits) - 30D
Hex signed 2's complement (4 digits) - 030D

**Comment by peterbaccaro** [comment]
The longest war in history is believed to be the Reconquista (Spanish for Reconquest), with a duration of 781 years which in hex is **30D** years.

**Comment by nsafarova** [comment]
The longest war in history is debatable, as there are many conflicts that could be considered "wars" and have lasted for varying lengths of time. However, one conflict that is often cited as the longest war in recorded history is the Reconquista, which lasted for approximately 781 years.

781 in decimal is 30D in hexadecimal. Therefore, the answer to your question is 30D.
